### ISSUE TITLE
Lack of Basic Docstrings in Mercury WIdget

### ISSUE THREAD

**Issue by ranggakd** [mljar/mercury#380]
## Description
I've noticed that the Mercury widget lacks basic docstrings. This makes it difficult to understand what each function does and how to use them properly on the go without going back and forward to the documentation page.

## Suggested Enhancement
Adding docstrings to the functions in the Mercury widget would greatly improve its usability and maintainability. These docstrings should include:

- A brief description of what the function does.
- A description of the function's parameters, including their types and what they represent.
- A description of the function's return value, if any.

## Comparison with `pandas`
Here's the comparison of docstring between `mr.Select` and `pd.DataFrame`:

`mr.Select()` and the [source code](https://github.com/mljar/mercury/blob/main/mercury/widgets/select.py)

![mercury](https://github.com/mljar/mercury/assets/36787606/53bb098d-698b-40ee-978a-dd1d9995cb3c)

`pd.DataFrame()` and the [source code](https://github.com/pandas-dev/pandas/blob/main/pandas/core/frame.py)

![pandas](https://github.com/mljar/mercury/assets/36787606/1072f4d0-23f3-4d06-a219-134965ebb0b6)

## Suggestion

I'm interested in contributing to this enhancement. I believe that adding these docstrings would greatly improve the usability and maintainability of the Mercury overall. I'm ready to start working on this as soon as I get your approval. Looking forward to contributing to this project!

**Comment by pplonski** [comment]
Good point @ranggakd. Ive assigned issue for you. In case any questions, Im happy to help.

**Comment by ranggakd** [comment]
Hello @pplonski, thank you for the opportunity to contribute to enhancing the Mercury widget docstrings. I'm eager to get started and want to ensure we're on the same page as it's my first time contributing to a large open-source project.

1. Widget Priority:
Would you prefer I start with the Input Widgets, Output Widgets, or both simultaneously? I'm inclined to begin with Input Widgets first because they're the first point of interaction, but I'm open to suggestions based on project priorities.

2. Commit Guidelines:
Without specific contribution guidelines, I plan to make small, focused commits for each change. Does that align with your expectations?

**Comment by pplonski** [comment]
Hi @ranggakd,

1. It's ok to start with input widgets.
2. Small commits - one per widgets are ok. 

Thank you!
